pkg/kademlia/kademlia_test.go

```diff
@@ -54,7 +58,7 @@ func TestNewKademlia(t *testing.T) {
 			}(),
 			bn:    []pb.Node{pb.Node{Id: "foo"}},
 			addr:  "127.0.0.1:8080",
-			setup: func() error { return os.RemoveAll("db") },
+			setup: func() error { return nil },
```

Comment:
This test was verifying that if the directory doesn't exist it will create it and not fail. We still need to delete the directory first here